src/robusta/cli/main.py

```diff
@@ -194,13 +196,24 @@ def gen_config(
     # asking the question
     if robusta_api_key is None:
         if typer.confirm(
-            "Would you like to use Robusta UI? This is HIGHLY recommended."
+            "Would you like to use Robusta UI? This is HIGHLY recommended.", default=True
         ):
             if typer.confirm("Do you already have a Robusta account?"):
-                robusta_api_key = typer.prompt(
-                    "Please insert your Robusta account token",
-                    default=None,
-                )
+                while True:
+                    robusta_api_key = typer.prompt(
+                        "Please insert your Robusta account token",
+                        default=None,
+                    )
+                    try:
+                        json.loads(base64.b64decode(robusta_api_key))
+                        break
+                    except Exception:
+                        typer.secho(
+                            "Sorry, invalid token format. "
+                            "The token can be found in your generated_values.yaml file, under the robusta_sink",
```

Comment:
Instead of "in your generated_values.yaml file" I think it's more clear to write one of the following (either is fine):

* "in any existing generated_values.yaml file"
* "by running `helm get values robusta`"

Reply:
fixing